app/services/bill_service.rb

```diff
@@ -10,10 +10,14 @@ def bills_by_query
     response = Rails.cache.fetch(bills_key) do
       fetch_bills_by_query_from_api
     end
-    json = JSON.parse(response.body, symbolize_names: true)
-    bill_search_result = json[:searchresult]
-
-    json
+  
+    if response&.body
+      json = JSON.parse(response.body, symbolize_names: true)
```

Comment:
should this say `if response.body` -- does it need the & ?

Reply:
I think its some sort of ruby operator that checks if its nil. could be wrong